app/src/main/java/ai/elimu/analytics/entity/StoryBookLearningEvent.java

```diff
@@ -10,6 +10,7 @@ public class StoryBookLearningEvent extends LearningEvent {
 
     @NonNull
     private Long storyBookId;
+    private Long storyBookTitle;
```

Comment:
```suggestion

    private String storyBookTitle;
```

Reply:
Ahh yes. Overlooked that, thanks for pointing it out !